src/site/content/en/blog/wakelock/index.md

```diff
@@ -69,7 +71,7 @@ Of course, there are plenty of other use cases:
 | 1. Create explainer                        | N/A                          |
 | 2. Create initial draft of specification   | [Complete][spec-ed]          |
 | **3. Gather feedback and iterate design**  | [**In Progress**](#feedback) |
```

Comment:
Should this line be unbolded since the API is now in the origin trial phase?

Reply:
Technically both states are active now. Not sure what’s the right thing to do here.

Reply:
Ah, OK. Seems fine then.